solver/src/settlement_submission/flashbots_settlement.rs

```diff
@@ -243,10 +243,8 @@ impl<'a> FlashbotsSolutionSubmitter<'a> {
                             err
                         );
 
-                        // if cancelation fails, we dont want to submit a new tx, rather contine and wait
-                        // for a nonce to change and end this function or a next cancelation to succeed.
-                        tokio::time::sleep(UPDATE_INTERVAL).await;
-                        continue;
+                        // if cancellation fails, we dont want to submit a new tx
```

Comment:
Nit: can you add more context to this comment on why this is needed (right now it just explains what the next line does)

Reply:
Fixed.